src/components/SocialLinks/SocialLinks.jsx

```diff
@@ -10,6 +10,11 @@ import { getSocialSettings } from '../../actions/getSocialSettings';
 import { Icon } from 'semantic-ui-react';
 import cx from 'classnames';
 
+// temporary fix for twitter-x icon
+import { Icon as IconCustom } from '@plone/volto/components';
```

Comment:
per coerenza chiamiamolo VoltoIcon

```suggestion
import { Icon as VoltoIcon } from '@plone/volto/components';
```